doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/02_Nginx_Configuration.md

```diff
@@ -109,7 +109,7 @@ server {
 
     # Some Admin Modules need this:
     # Database Admin, Server Info
-    location ~* ^/admin/(adminer|external) {
+    location ~* ^/admin/external {
```

Comment:
This probably needs to be adapted in the skeleton/demo, too.

Reply:
yes, here https://github.com/pimcore/demo/blob/11.x/.docker/nginx.conf#L94 & https://github.com/pimcore/skeleton/blob/11.x/.docker/nginx.conf#L94